clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -207,6 +202,18 @@
     public static final String CHECK_CRCS_CONFIG = "check.crcs";
     private static final String CHECK_CRCS_DOC = "Automatically check the CRC32 of the records consumed. This ensures no on-the-wire or on-disk corruption to the messages occurred. This check adds some overhead, so it may be disabled in cases seeking extreme performance.";
 
+    /**
+     * <code>list.key.serializer.inner</code>
+     */
+    public static final String LIST_KEY_DEERIALIZER_INNER_CLASS_CONFIG = "list.key.deserializer.inner";
+    public static final String LIST_KEY_DESERIALIZER_INNER_CLASS_DOC = "Inner deserializer class for key that implements the <code>org.apache.kafka.common.serialization.Deserializer</code> interface.";
```

Comment:
We should point out, that this config is only affective iff `key.deserializer` is set to `ListDeserializer`.